### ISSUE TITLE
Old version

### ISSUE THREAD

**Issue by DavidBE1** [venomous0x/WhatsAPI#1108]
Hi 
I get the Old version message when trying to get code
{"status":"fail","reason":"old_version"}
Read and did all what was writen before but still no pogress
Can someone help ?

thanks

David

**Comment by mgp25** [comment]
@DavidBE1 this repo is inactive, use WhatsAPI Official instead: https://github.com/mgp25/WhatsAPI-Official

And full documentation in the wiki: https://github.com/mgp25/WhatsAPI-Official/wiki

Regards

**Comment by DavidBE1** [comment]
Thanks,
That is a php project, is there no .net project any more ?

Thanks
David

**Comment by mgp25** [comment]
@DavidBE1 Yes, but both are inactive, use shirioko one, take the token we are using in WhatsAPI Official and use it in the .NET API
